### PR TITLE
pyaez package added to the conda forge

### PR DESCRIPTION
**Pull request by iamtekson**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by iamtekson** [comment]
@conda-forge-admin, please ping the team

**Comment by iamtekson** [comment]
@conda-forge/staged-recipes, this one is ready for review.

File: recipes/example/meta.yaml [hunk 1]
```diff
@@ -92,4 +92,4 @@ extra:
     # GitHub IDs for maintainers of the recipe.
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
     - LisaSimpson
-    - LandoCalrissian
+    - LandoCalrissian
```
Comment (ocefpaf) [inline recipes/example/meta.yaml:95]: Can you revert the change in our example recipe? If that is complicated for you I can do it for you, just let me know.

File: recipes/pyaez/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/pyaez/meta.yaml:1]: Ping! Also, try to use grayskull to create your recipes, see https://github.com/conda-incubator/grayskull

File: recipes/example/meta.yaml (hunk 1 above)
Comment (iamtekson) [reply recipes/example/meta.yaml:95]: Please, you can proceed!

Comment (ocefpaf) [reply recipes/example/meta.yaml:95]: Looks like you do not have a "let maintainers edit" marked on your PR. You can do that with `git co 4e6727a61e7cb82d5db2d3171ba460e908c0ead7 -- meta.yaml` and then commit it.

BTW, I'd address the recipe you are submitting first.

Comment (iamtekson) [reply recipes/example/meta.yaml:95]: Sorry, that was my mistake. I unchecked this button.

File: recipes/pyaez/meta.yaml (hunk 2 above)
Comment (iamtekson) [reply recipes/pyaez/meta.yaml]: I just removed the comments as well.

File: recipes/pyaez/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  imports:
+    - pyaez
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/gicait/pyaez
+  summary: 'PyAEZ is a python package consisted of many algorithms related to Agro-ecalogical zoning (AEZ) framework. PyAEZ tries to encapsulate all complex calculations in AEZ and try to provide user friendly, and intuitive ways to input data and output results after calculations.'
```
Comment (ocefpaf) [inline recipes/pyaez/meta.yaml:39]: The summary is as big, actually the same, as the description. Upstream has it as:

```
PyAEZ is a python package consisted of many algorithms related to Agro-ecalogical zoning (AEZ) framework.
```

File: recipes/example/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipes/example/meta.yaml:95]: Still cannot edited it. You'll have to fix this one yourself or open a new PR.

File: recipes/pyaez/meta.yaml (hunk 3 above)
Comment (iamtekson) [reply recipes/pyaez/meta.yaml]: I just make it shorter.

File: recipes/example/meta.yaml (hunk 1 above)
Comment (iamtekson) [reply recipes/example/meta.yaml]: I just revert the `example/meta.yaml` file. I hope now it is okay to merge this PR.

**Pull request merged**